infrastructure/reftest/reftest_match-print-ref.html

```diff
@@ -0,0 +1,13 @@
+<title></title>
+<meta name="reftest-pages" content="1-2">
+<style>
+* {margin: 0, padding:0}
+/* It is unclear to me why this is 180px rather than 192px (2 inches at 96dpi) */
+div {height:180px;}
```

Comment:
Where does this 2in come from? But my first thought would be "what's the margin/padding on `@page`?".

Reply:
Page is 5*3 inches with 0.5 inch margins.

Reply:
Yeah I would try `@page { margin: 0; }` and see if that fixes it.